src/by/epam/nickgrudnitsky/entity/problem/BirthdayProblem.java

```diff
@@ -0,0 +1,78 @@
+package by.epam.nickgrudnitsky.entity.problem;
+
+import by.epam.nickgrudnitsky.entity.output.Printer;
+import by.epam.nickgrudnitsky.util.Validator;
+
+import java.util.HashSet;
+import java.util.Random;
+import java.util.Set;
+
+import static by.epam.nickgrudnitsky.util.DataInput.readNumber;
+import static by.epam.nickgrudnitsky.util.Validator.validateNumberOfChecks;
+import static by.epam.nickgrudnitsky.util.Validator.validateNumberOfPeople;
+
+public class BirthdayProblem extends Problem {
+    private int numberOfPeople;
+    private int numberOfChecks;
+    private int numberOfCoincidences;
+    private Set<Integer> birthdays = new HashSet<>();
+    private static Random random = new Random();
+    private final static int DAYS_IN_A_YEAR = 365;
```

Comment:
https://github.com/Yauhenda/task_01/pull/1#discussion_r379362906 was not about line by line order but about word by word order. So `private final static` > `private static final`
Please refer to https://docs.oracle.com/javase/specs/jls/se7/html/jls-8.html#jls-8.3.1